README.md

```diff
@@ -9,6 +9,52 @@
 
 `composer require lullabot/mpx-php`
 
+## Example
```

Comment:
I think it would be nice to explain a little bit what is this example about, before all the following code